tracer/test/Datadog.Trace.Security.Unit.Tests/WafErrorsTests.cs

```diff
@@ -7,28 +7,26 @@
 using System.Collections.Specialized;
 using Datadog.Trace.AppSec;
 using Datadog.Trace.AppSec.Waf;
-using Datadog.Trace.AppSec.Waf.NativeBindings;
-using Datadog.Trace.Configuration;
+using Datadog.Trace.Security.Unit.Tests.Utils;
 using FluentAssertions;
 using Xunit;
 
 namespace Datadog.Trace.Security.Unit.Tests
 {
     [Collection("WafTests")]
```

Comment:
Are the `[Collection]` attributes needed or not? 🤔 If yes, we should have them in _every_ class that needs to run serially, e.g. `EncoderUnitTests`?. Maybe we should add it to the `WafLibraryRequiredTest` class (along with `[CollectionDefinition]`)

Reply:
removed in a13e5682a8421187c2a72718420e9b5f19ba8318 I think they can now run parallel